src/components/CustomRegisterField/CustomRegisterField.tsx

```diff
@@ -0,0 +1,71 @@
+import { type FC, type ChangeEventHandler, type ReactNode, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import type { GetRegisterFieldOption } from '@inplayer-org/inplayer.js';
+
+import Checkbox from '#components/Checkbox/Checkbox';
+import TextField from '#components/TextField/TextField';
+import Radio from '#components/Radio/Radio';
+import Dropdown from '#components/Dropdown/Dropdown';
+import DateField from '#components/DateField/DateField';
+import { ConsentFieldVariants, REGISTER_FIELD_VARIANT } from '#src/services/inplayer.account.service';
+import countriesCodes from '#static/countries-codes.json';
```

Comment:
It looks like this is the only place this import is used... it's probably not worth adding an alias to save 3 characters on 2 lines in a single file.
```suggestion
import countriesCodes from '#src/static/countries-codes.json';
```